UPGRADE-3.4.md

```diff
@@ -4,6 +4,10 @@ UPGRADE FROM 3.3 to 3.4
 DependencyInjection
 -------------------
 
+ * Autowiring service auto-registration is deprecated and won't be supported
+   in Symfony 4.0. Explicitely inject your dependencies or create services
+   whose ids are their fully-qualified class name.
```

Comment:
An example with `before`/`after` would probably help people understand what that means. For anybody not familiar with the container, this is cryptic :)